### PR TITLE
fix: save news

### PR DESCRIPTION
**Pull request by bangunbagustapa**
### Changes

- Fix tags data format when submitting the form
- Fix loading stuck when error uploading image

### Evidence

Title: fix: tags data format
Project: Portal Jabar
Participants: @bangunbagustapa @naufalihsank @Ibwedagama @yoslie

### CODE REVIEW

**Review by yoslie — APPROVED** [review]

File: src/pages/News/CreateEditNews.vue [hunk 1]
```diff
@@ -840,11 +840,12 @@ export default {
...
         try {
+          this.progress = 50;
           image = await this.uploadMedia(image);
         } catch (error) {
+          this.progress = 100;
+          this.loading = false;
           this.setMessage('ERROR', 'Gagal menyimpan berita', 'Terjadi kesalahan dalam menyimpan berita');
-        } finally {
-          this.progress = 50;
         }
```
Comment (Ibwedagama) [inline src/pages/News/CreateEditNews.vue:849]: ask:

hello @bangunbagustapa , when the upload media is successful, where is the mutation of `this.progess` ? is it automatically changed to 100? 🤔

Comment (bangunbagustapa) [reply src/pages/News/CreateEditNews.vue:849]: Yes, when we will about to upload, the progress is 50, and when the upload success, the progress is 100 right before submitting the form. Wdyt?

Comment (bangunbagustapa) [reply src/pages/News/CreateEditNews.vue:849]: The mutation for progress is inside `saveNews` action

Comment (Ibwedagama) [reply src/pages/News/CreateEditNews.vue:849]: Oh, I see, I didn't see that 😅 

any reason `this.progress` mutation is on another function? or is it the desired behavior? @bangunbagustapa

Comment (bangunbagustapa) [reply src/pages/News/CreateEditNews.vue:849]: So, my idea is whenever the process touch the submit action, i want it to be 100.

So 20 if the user click the button, 50 when the image has to be uploaded first (if we don't have to be uploaded, then jump to the next progress), 100 when the data ready to be submitted.

Comment (Ibwedagama) [reply src/pages/News/CreateEditNews.vue:849]: Oh I see, okay then approved @bangunbagustapa 👍🏼

**Review by Ibwedagama — APPROVED** [review]

**Review by naufalihsank — APPROVED** [review]

**Pull request merged**